investment-time/README.md

```diff
@@ -0,0 +1,81 @@
+# Investment Time
+
+This directory contains documentation of DataMade's implementation of investment
+time, as well as pointers to documents related to the proposal, pilot, and
+adoption of investment time.
+
+## Contents
+
+- README – You are here! 🎉
+- [Research](./research.md)
+
+## Investment Time at DataMade
+
+### How much investment time do I have?
+
+DataMade employees may use 10% of hours worked in a given two-week cycle, or
+one of every 10 days worked, for investment time.
+
+Employees will be assigned to one of two cohorts with alternating two-week
+cycles, so that investment time is staggered across the team. Assignments will
+appear on the DataMade calendar as `${STAFF} investment time`, every other
+Friday.
+
+Staff may use investment time as their schedule allows during the week of their
+designated investment day, however it may not be used during their off week.
```

Comment:
In thinking more about scheduling / billing, I think that it would be best to limit use of investment time to the week of staff's designated investment day, that way we don't have weeks where 3-4 of us are using investment time / not logging billable hours. In my mind, our weekly planning process (in general) gives us the structure and information we need to plan investment time accordingly, and as managers, we can help to protect staff time so unplanned work does not interfere with investment time. @jeancochrane, as someone who has been really proactive about rescheduling missed investment time, do you think this makes sense, or does it just impose a barrier to using the time?

Reply:
This makes sense to me if weekly billing is a critical metric for us. I'll note that I would expect this policy to make it much harder for me to reschedule investment time -- I used to only schedule it every other week and I was pretty much unable to successfully reschedule time back then, since it required me to take even more client time out of investment weeks, which were already constrained. Since adopting a practice of just pushing the day back a week if necessary I've been more successful at rescheduling time. So I think there's a tradeoff here that depends on how important our weekly billing is to us vs. how important investment time is.

Reply:
Hm, good point. It's my understanding that monthly, not weekly, billing is the more important metric to us, so at that scale, rescheduling time should have little effect on billing at the month scale.

More specifically, investment time can go unused for two reasons:

1. Time off
2. Other obligations (usually client work) taking precedence

In the first case, time off, both billable hours and investment time are _proportions_ of time worked, not constants. So, supposing your job description stipulates 65% of time is spent on client work and 10% on investment time, so long as you're spending the time you do work in those proportions month to month, I don't suppose it matters when investment time is taken!

In the second case, other obligations, _more_ billable hours than planned are logged the week investment time is not taken, so if that investment time is moved to the following week, billable hours for the two-week period (and, by extension, the month) should be net neutral. (This is logically related to the first point.)

It seems as though billing is not a compelling reason to limit the scheduling of investment time. This makes me return to why we are staggering time across alternating weeks at all, rather than using the ["20% time" model](https://en.wikipedia.org/wiki/20%25_Project). For me, it comes down to minimizing unplanned work and protecting / mutually respecting investment time. thoughtbot [puts it this way](https://thoughtbot.com/blog/investment-time):

> The primary problem is that if no one knows when their colleagues’ 20% is assigned, it’s difficult to mutually respect it. The perceived urgency of tasks is hard to measure and the time becomes unprotected.

> By assigning the time to a single day, everyone’s expectations are set.

So, I'm curious: What are the barriers to taking investment time in a given week? For me, it's definitely a sense of obligation to finish client work, usually in excess of what I had planned. What about you? Is there a way we could change our thinking / adjust our expectations so that investment time is protected?

Reply:
> This makes me return to why we are staggering time across alternating weeks at all, rather than using the "20% time" model. For me, it comes down to minimizing unplanned work and protecting / mutually respecting investment time.

These are compelling reasons to me, but wouldn't these motivations suggest that we try to consolidate investment days, rather than stagger them?

> What are the barriers to taking investment time in a given week?

Some factors that I've found affect my own investment time:

1. Management duties (someone needs help getting unstuck, or needs new tasks)
2. Client work deadlines
3. Having a lot of projects (more projects -> more deadlines)
4. Working reduced hours (weeks contain fewer hours, but I have a similar amount of responsibility)

Of these, I think 1) could be mitigated by trying harder to consolidate our investment time on the same days. I could address 3) and 4) by trying to be more aggressive about restricting the scope of my responsibility. 2) seems like the trickiest one, as you've identified, and is also the one that Thoughtbot seems to call out as pretty much inevitable. Curious to hear what you think!

Reply:
Thank you for the thoughtful comment, @jeancochrane! Taking these points into consideration, while also thinking of our demonstrated pattern of investment time use, I'm really interested in the idea of a consolidated investment day, with some flexibility for rescheduling _during that week_ (but not during off weeks).

I agree that client deadlines are an inevitable challenge. Do you find that they're consistently barriers to using investment time, or only occasionally? Would accounting for investment days when developing project timelines help to mitigate deadline pressures? One thing that thoughtbot seems to do well is set expectations with their clients – do we need to do any work to set or maintain similar boundaries with our clients?

> 4. Working reduced hours (weeks contain fewer hours, but I have a similar amount of responsibility)

This really grabs my attention! I think we have some work to do internally to separate responsibility from productivity. Taking you and me as an example, it makes sense that we have the same _kinds_ of work, but it's just not reasonable to expect that we produce the same _volume_ of output, when you have one less day! Is there anything I can do to help relieve some of this pressure? What about at the organization level (as I'm sure this impacts Bea / any other hourly staff we may have in the future, as well)?

Reply:
(P.s., If we were to constrain makeup time to the week of the designated investment day, would scheduling investment day midweek, instead of at the end of the week, make it easier to make up missed time?)

Reply:
> I'm really interested in the idea of a consolidated investment day, with some flexibility for rescheduling during that week (but not during off weeks)... I agree that client deadlines are an inevitable challenge. Do you find that they're consistently barriers to using investment time, or only occasionally? Would accounting for investment days when developing project timelines help to mitigate deadline pressures?

I think it's definitely worth trying. One idea that could make sense to try might be to consolidate investment days on Thursdays and then have a policy of no deadlines on Thursdays/Fridays (it might even make sense to exclude Mondays too). 

> Is there anything I can do to help relieve some of this pressure? What about at the organization level (as I'm sure this impacts Bea / any other hourly staff we may have in the future, as well)?

This is a really good question, and my thoughts are unfortunately not fully developed yet. But the first thing that comes to mind is that we don't really have any principled guidelines (to my knowledge) on the appropriate number of projects for employees at different levels and with different weekly hours to take on. I suspect that I tend to take on too many without fully understanding the time commitments I'm getting myself into. Another thing I'll add is that our 1on1s often serve as an ad hoc way of dealing with this problem, where you and I can compare our workloads and try to relieve pressure. I feel like it's worked well so far, and periods where I've taken less investment time have sometimes been periods where I've intentionally taken on more work than usual in order to help take things off your plate.

Reply:
what's the status of your recommendation here, @hancush? what feedback are you looking for from partners?